// html-validate/build/shim.js
"use strict";
/* used when calling require('htmlvalidate'); */
Object.defineProperty(exports, "__esModule", { value: true });
var htmlvalidate_1 = require("./htmlvalidate");
exports.HtmlValidate = htmlvalidate_1.default;
var cli_1 = require("./cli/cli");
exports.CLI = cli_1.CLI;
var config_1 = require("./config");
exports.Config = config_1.Config;
exports.ConfigError = config_1.ConfigError;
exports.ConfigLoader = config_1.ConfigLoader;
exports.Severity = config_1.Severity;
var dom_1 = require("./dom");
exports.DynamicValue = dom_1.DynamicValue;
exports.HtmlElement = dom_1.HtmlElement;
exports.NodeClosed = dom_1.NodeClosed;
exports.TextNode = dom_1.TextNode;
var error_1 = require("./error");
exports.UserError = error_1.UserError;
var rule_1 = require("./rule");
exports.Rule = rule_1.Rule;
var reporter_1 = require("./reporter");
exports.Reporter = reporter_1.Reporter;
var transform_1 = require("./transform");
exports.TemplateExtractor = transform_1.TemplateExtractor;
var parser_1 = require("./parser");
exports.Parser = parser_1.Parser;
const pkg = require("../package.json");
exports.version = pkg.version;
